src/cursor.rs

```diff
@@ -165,8 +152,53 @@ impl fmt::Display for BadImage {
 
 impl Error for BadImage {}
 
-/// Platforms export this directly as `PlatformCustomCursor` if they need to only work with images.
-#[derive(Debug, Clone, PartialEq, Eq)]
+/// Platforms export this directly as `PlatformCustomCursorBuilder` if they need to only work with images.
+#[derive(Debug)]
+pub struct OnlyCursorImageBuilder(pub CursorImage);
```

Comment:
Shouldn't the inner field be `pub(crate)`?

```suggestion
pub struct OnlyCursorImageBuilder(pub(crate) CursorImage);
```

Reply:
`OnlyCursorImageBuilder` isn't actually exposed through the library.
I would generally prefer if we applied [`unreachable_pub`](https://doc.rust-lang.org/1.74.0/rustc/lints/listing/allowed-by-default.html#unreachable-pub) on the whole library.

But to avoid confusion I will mark `OnlyCursorImageBuilder` itself as `pub(crate)`.

Reply:
Done.